### PR TITLE
[JENKINS-34848] Add gradle installation.

### PR DESCRIPTION
**Pull request by andresrc**
See [JENKINS-34848](https://issues.jenkins-ci.org/browse/JENKINS-34848).

@reviewbybees

### CODE REVIEW

**Comment by ghost** [comment]
This pull request originates from a [CloudBees](https://www.cloudbees.com/) employee. At CloudBees, we require that all pull requests be reviewed by other CloudBees employees before we seek to have the change accepted. If you want to learn more about our process please see [this explanation](https://github.com/reviewbybees/about#about-reviewbybees).

File: src/main/java/org/jvnet/hudson/test/ToolInstallations.java [hunk 1]
```diff
@@ -129,6 +131,33 @@
...
+    /**
+     * Extracts Gradle and configures it.
+     */
+    public static GradleInstallation configureDefaultGradle(TemporaryFolder tmp) throws Exception {
+        GradleInstallation installation;
+        if (System.getenv("GRADLE_HOME") != null) {
+            installation = new GradleInstallation("default", System.getenv("GRADLE_HOME"), JenkinsRule.NO_PROPERTIES);
+        } else {
+            LOGGER.warning("Extracting a copy of Gradle bundled in the test harness. "
+                    + "To avoid a performance hit, set the environment variable GANT_HOME to point to an  Ant installation.");
```
Comment (armfergom) [inline src/main/java/org/jvnet/hudson/test/ToolInstallations.java]: `GANT_HOME` or `GRADLE_HOME`? Also, "(...) point to an **Ant installation**."? Shouldn't it say "Gradle installation" instead?

Comment (andresrc) [inline src/main/java/org/jvnet/hudson/test/ToolInstallations.java]: the joys of copy & paste...

**Comment by aheritier** [comment]
@jglick @jenkinsci/code-reviewers I think this one is ok to be merged

**Comment by jglick** [comment]
:bee:

**Comment by jglick** [comment]
Please consider amending as per #5 to not store binaries in the Git repository.

**Pull request merged**